pkg/olm/olm.go

```diff
@@ -272,15 +272,9 @@ func GenerateCSV(opConf *operator.Conf) *operv1.ClusterServiceVersion {
 				Description:  "PVPoolDefaultStorageClass (optional) overrides the default cluster StorageClass for the pv-pool volumes. This affects where the system stores data chunks (encrypted). Updates to this field will only affect new pv-pools, but updates to existing pools are not supported by the operator.",
 				DisplayName:  "PV Pool DefaultStorageClass",
 			},
-			operv1.SpecDescriptor{
```

Comment:
@dannyzaken Instead of deleting can you leave it commented out with a comment that explains the problem?